### PR TITLE
Add support button and popup 

### PR DESCRIPTION
**Pull request by lilia1891**
Related Asana tasks https://app.asana.com/0/1205310117865974/1207430612924933/f and https://app.asana.com/0/1205310117865974/1207430612924934/f

Screen Shot 2024-06-20 at 09 50 38

Screen Shot 2024-06-20 at 15 09 43

### CODE REVIEW

File: components/AppBar.tsx [hunk 1]
```diff
@@ -51,80 +53,101 @@ export function AppBar() {
...
+          </Stack>
+
+          <div>
+            <Stack direction="row" spacing={2}>
+              <Button
+                size="small"
+                variant="text"
+                color="inherit"
+                onClick={handleOpenSupportModal}
+                endIcon={<InfoCircle size={22} />}
```
Comment (woodwoerk) [inline components/AppBar.tsx:101]: Let's go with the `QuestionCircle` icon instead, this bootstrap icon doesn't quite fit

https://icons.getbootstrap.com/icons/question-circle/

File: components/SupportModal.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+const supportData = {
+  imageSrc: '/images/nzc_img.png',
+  title: 'Welcome to NetZeroPaths',
+  content:
+    'Thanks for joining us on this journey towards a sustainable future for your city. Here are three quick tips to help you get started!',
+  getStartedUrl: '/welcome',
+};
+
+const SupportModal: React.FC<SupportModalProps> = ({ open, onClose }) => {
```
Comment (woodwoerk) [inline components/SupportModal.tsx:29]: I remember reading a while ago that using `React.FC` has no benefits and is more verbose, do you have any thoughts on this? I can try to dig up where I read it

Comment (woodwoerk) [reply components/SupportModal.tsx:29]: > React.FC is unnecessary: it provides next to no benefits and has a few downsides. (See below.)
https://github.com/facebook/create-react-app/pull/8177

I don't have a super strong opinion on this, but perhaps for consistency with can stick with the `const SupportModal = ({ open, onClose }: SupportModalProps) => { ` approach for consistency and simplicity

**Review by woodwoerk — APPROVED** [review]
Looks good, added a few small comments

File: components/SupportModal.tsx (hunk 2 above)
Comment (lilia1891) [reply components/SupportModal.tsx:29]: Yes, sure. Good to know, thanks

**Pull request merged**